index.bs

```diff
@@ -7914,10 +7904,25 @@ Constructors</h4>
 				[[mediacapture-streams#mediastream|MediaStream]] that has at least one
 				[[mediacapture-streams#mediastreamtrack|MediaStramTrack]] whose
 				<code>kind</code> attribute has the value <code>"audio"</code>,
-				throw an {{InvalidStateError}} and abort these steps.
-			3. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
+				throw an {{InvalidStateError}} and abort these steps. Else, let
+				this stream be <var>inputStream</var>.
+			3. Let <var>tracks</var> the list of all
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] of
+				<var>inputStream</var> that have a <code>kind</code> of
+				<code>"audio"</code>.
+			4. Sort the elements in <var>tracks</var> based on their <code>id</code>
+				attribute, ordering by Unicode code point
+			5. Let the first element of <var>tracks</var> be the
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] used as the
+				input audio for this {{MediaStreamAudioSourceNode}}.
+			6. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
 				object. <a href="#audionode-constructor-init">Initialize</a>
 				<var>node</var>, and return <var>node</var>.
+
+			Note: The behaviour for picking the track to output is weird for legacy
```

Comment:
I am not sure what this means. Can you elaborate?

Also perhaps: s/weird/inappropriate/

Reply:
I just copy/pasted this from above. I'm going to use the word "arbitrary", I think it reflects the thinking here and is more neutral.

Reply:
Would "unspecified" be too strong for this? So what we want to point out is there might be an interop issue.

Reply:
We can't say it's unspecified if we're speccing it. We know what is going to happen, without ambiguity, it's just a bit strange.